### DISCUSSION TITLE
Question About Redirect to Login for Screens With Rule Requirement

### DISCUSSION THREAD

**Discussion by Fierce-Cat** [Budibase/budibase#6796] [Help]
Hi guys,

I just upgraded to the brand new v1.1.13 (from 1.0.110, yeah, pretty old) and saw some awesome redesigns and new features, thanks to the team and community for all the wonderful work. But I noticed that if I redirect to a screen that requires permissions (like Basic) in the navigation or with a button on the home page, the page will stay at the current path, and if a new window is opened with that screen path, it will redirect back to the home page. 

I have not encountered this problem in the old version, which Budibase will automatically redirect users to login page.

Is this a bug? Or some new permission design?

I've created a sample app, you can click [PageWithBasicRule](https://yixnet.budibase.app/app/test#/basic) on the navigation to see the problem: [https://yixnet.budibase.app/app/test#/home](https://yixnet.budibase.app/app/test#/home)

Example:
![example](https://s2.loli.net/2022/07/19/wGg9EYCUcNyken3.gif)

**Comment by melohagan** [comment]
Hi @Fierce-Cat 

Thanks for kind words! 

Would you be able to share a screenshot of the navigation links configuration please?

**Reply by Fierce-Cat** [reply]
Of cource!
![image](https://user-images.githubusercontent.com/21760321/181171331-53322845-a9c9-42cc-85a2-66098f0b97ef.png)

**Comment by dewdeded** [comment]
Any news?

**Reply by melohagan** [reply]
Hey @dewdeded Is your use-case the same? I'm not entirely sure if this is intended behavior or not. Depends on what you're looking to do.
